test/integ/main_test.go

```diff
@@ -244,6 +245,7 @@ func TestJVS(t *testing.T) {
 	}
 }
 
+//nolint:tparallel // subtests need to run in sequence.
```

Comment:
Can you elaborate on why?

Reply:
It was written in https://github.com/abcxyz/jvs/blob/23cdda185a8fc277ba798bfaf3e34543458f1934/test/integ/main_test.go#L280
Should I move the original comment to here to make it clear?

Reply:
Yes, please

Reply:
done